### ISSUE TITLE
Bug: Port Forwarding is broken with PIA after latest update

### ISSUE THREAD

**Issue by SimplicityGuy** [qdm12/gluetun#576]
**Is this urgent?**: No

**Host OS** (approximate answer is fine too): docker container on Debian 11/bullseye

**CPU arch** or **device name**: amd64

**What VPN provider are you using**: PIA

**What are you using to run your container?**: Docker Compose

**What is the version of the program** built on 2021-08-20T01:16:54Z (commit 8d258fe)

**What's the problem** 🤔

Port Forwarding has stopped working with this update.

```log

Running version latest built on 2021-08-20T01:16:54Z (commit 8d258fe)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2021/08/21 05:15:08 INFO Alpine version: 3.14.1
2021/08/21 05:15:08 INFO OpenVPN 2.4 version: 2.4.11
2021/08/21 05:15:08 INFO OpenVPN 2.5 version: 2.5.2
2021/08/21 05:15:08 INFO Unbound version: 1.13.1
2021/08/21 05:15:08 INFO IPtables version: v1.8.7
2021/08/21 05:15:08 INFO Settings summary below:
|--VPN:
   |--Type: openvpn
   |--OpenVPN:
      |--Version: 2.5
      |--Verbosity level: 1
      |--Network interface: tun0
      |--Run as root: enabled
      |--Encryption preset: strong
   |--Private Internet Access settings:
      |--Regions: netherlands
      |--OpenVPN selection:
         |--Protocol: udp
         |--PIA encryption preset: strong
      |--Port forwarding:
         |--File path: /tmp/gluetun/forwarded_port
|--DNS:
   |--Plaintext address: 1.1.1.1
   |--DNS over TLS:
      |--Unbound:
          |--DNS over TLS providers:
              |--Quad9
              |--Quadrant
          |--Listening port: 53
          |--Access control:
              |--Allowed:
                  |--0.0.0.0/0
                  |--::/0
          |--Caching: enabled
          |--IPv4 resolution: enabled
          |--IPv6 resolution: disabled
          |--Verbosity level: 1/5
          |--Verbosity details level: 0/4
          |--Validation log level: 0/2
          |--Username:
      |--Blacklist:
         |--Blocked categories:
         |--Additional IP networks blocked: 13
      |--Update: every 24h0m0s
|--Firewall:
|--System:
   |--Process user ID: 1000
   |--Process group ID: 1000
   |--Timezone: NOT SET ⚠️ - it can cause time related issues
|--Health:
   |--Server address: 127.0.0.1:9999
   |--OpenVPN:
      |--Initial duration: 6s
      |--Addition duration: 5s
|--HTTP control server:
   |--Listening port: 8000
   |--Logging: enabled
|--Public IP getter:
   |--Fetch period: 12h0m0s
   |--IP file: /tmp/gluetun/ip
|--Github version information: enabled


```

**Comment by qdm12** [comment]
Hi there, what do you mean that it has stopped? Do you mean it doesn't trigger (no logs) at all? I'll check, thanks for the feedback

**Comment by camcop** [comment]
Hi,

I believe I have the same issue with port forwarding with PIA on Raspberry Pi 4. Can't see any mention of port forwarding in the logs.

Apologies for the line spacing, not sure how to fix this.

`
2021/08/21 14:07:35 INFO Settings summary below:

|--VPN:

   |--Type: openvpn

   |--OpenVPN:

      |--Version: 2.5

      |--Verbosity level: 1

      |--Network interface: tun0

      |--Run as root: enabled

      |--Encryption preset: strong

   |--Private Internet Access settings:

      |--Regions: uk london

      |--OpenVPN selection:

         |--Protocol: udp

         |--PIA encryption preset: strong

      |--Port forwarding:

         |--File path: /srv/dev-disk-by-label-Data1/appdata/gluetun/forwarded_port

|--DNS:

   |--Plaintext address: 1.1.1.1

   |--DNS over TLS:

      |--Unbound:

          |--DNS over TLS providers:

              |--Cloudflare

          |--Listening port: 53

          |--Access control:

              |--Allowed:

                  |--0.0.0.0/0

                  |--::/0

          |--Caching: enabled

          |--IPv4 resolution: enabled

          |--IPv6 resolution: disabled

          |--Verbosity level: 1/5

          |--Verbosity details level: 0/4

          |--Validation log level: 0/2

          |--Username: 

      |--Blacklist:

         |--Blocked categories: malicious

         |--Additional IP networks blocked: 9

      |--Update: every 24h0m0s
      
|--Firewall:

   |--Outbound subnets: 192.168.8.0/24

|--System:

   |--Process user ID: 1000

   |--Process group ID: 1000

   |--Timezone: europe/london

|--Health:

   |--Server address: 127.0.0.1:9999

   |--OpenVPN:

      |--Initial duration: 6s

      |--Addition duration: 5s

|--HTTP control server:

   |--Listening port: 8000

   |--Logging: enabled

|--Public IP getter:

   |--Fetch period: 12h0m0s

   |--IP file: /tmp/gluetun/ip

|--Github version information: enabled

2021/08/21 14:07:36 INFO storage: merging by most recent 11007 hardcoded servers and 11007 servers read from /gluetun/servers.json

2021/08/21 14:07:36 INFO routing: default route found: interface eth0, gateway 172.17.0.1

2021/08/21 14:07:36 INFO routing: local ethernet link found: eth0

2021/08/21 14:07:36 INFO routing: local ipnet found: 172.17.0.0/16

2021/08/21 14:07:36 INFO routing: default route found: interface eth0, gateway 172.17.0.1

2021/08/21 14:07:36 INFO routing: adding route for 0.0.0.0/0

2021/08/21 14:07:36 INFO firewall: firewall disabled, only updating allowed subnets internal list

2021/08/21 14:07:36 INFO routing: default route found: interface eth0, gateway 172.17.0.1

2021/08/21 14:07:36 INFO routing: adding route for 192.168.8.0/24

2021/08/21 14:07:36 INFO TUN device is not available: open /dev/net/tun: no such file or directory; creating it...

2021/08/21 14:07:36 INFO firewall: enabling...

2021/08/21 14:07:36 INFO firewall: enabled successfully

2021/08/21 14:07:36 INFO dns over tls: using plaintext DNS at address 1.1.1.1

2021/08/21 14:07:36 INFO http server: listening on :8000

2021/08/21 14:07:36 INFO healthcheck: listening on 127.0.0.1:9999

2021/08/21 14:07:36 INFO firewall: setting VPN connection through firewall...

2021/08/21 14:07:36 INFO healthcheck: healthy!

2021/08/21 14:07:36 INFO vpn: 2021-08-21 14:07:36 DEPRECATED OPTION: ncp-disable. Disabling cipher negotiation is a deprecated debug feature that will be removed in OpenVPN 2.6

2021/08/21 14:07:36 INFO vpn: OpenVPN 2.5.2 armv7-alpine-linux-musleabihf [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on May  4 2021

2021/08/21 14:07:36 INFO vpn: library versions: OpenSSL 1.1.1k  25 Mar 2021, LZO 2.10

2021/08/21 14:07:36 INFO vpn: CRL: loaded 1 CRLs from file -----BEGIN X509 CRL-----

2021/08/21 14:07:36 INFO vpn: [REDACTED]

2021/08/21 14:07:36 INFO vpn: -----END X509 CRL-----

2021/08/21 14:07:36 INFO vpn: TCP/UDP: Preserving recently used remote address: [AF_INET][REDACTED]

2021/08/21 14:07:36 INFO vpn: UDP link local: (not bound)

2021/08/21 14:07:36 INFO vpn: UDP link remote: [AF_INET]212.102.63.154:1197

2021/08/21 14:07:36 INFO vpn: [london402] Peer Connection Initiated with [AF_INET][REDACTED]

2021/08/21 14:07:36 INFO vpn: TUN/TAP device tun0 opened

2021/08/21 14:07:36 INFO vpn: /sbin/ip link set dev tun0 up mtu 1500

2021/08/21 14:07:36 INFO vpn: /sbin/ip link set dev tun0 up

2021/08/21 14:07:36 INFO vpn: /sbin/ip addr add dev tun0 [REDACTED]

2021/08/21 14:07:36 INFO vpn: Initialization Sequence Completed

2021/08/21 14:07:36 INFO vpn: VPN routing IP address: 212.102.63.154

2021/08/21 14:07:36 INFO dns over tls: downloading DNS over TLS cryptographic files

2021/08/21 14:07:38 INFO dns over tls: downloading hostnames and IP block lists

2021/08/21 14:07:39 INFO dns over tls: init module 0: validator

2021/08/21 14:07:39 INFO dns over tls: init module 1: iterator

2021/08/21 14:07:39 INFO dns over tls: start of service (unbound 1.13.1).

2021/08/21 14:07:39 INFO dns over tls: generate keytag query _ta-4a5c-4f66. NULL IN

2021/08/21 14:07:39 INFO dns over tls: generate keytag query _ta-4a5c-4f66. NULL IN

2021/08/21 14:07:39 INFO dns over tls: ready

2021/08/21 14:07:40 INFO vpn: You are running on the bleeding edge of latest!

2021/08/21 14:07:40 ERROR vpn: cannot obtain VPN local gateway IP: VPN local gateway IP address not found: in 7 routes

2021/08/21 14:07:40 INFO ip getter: Public IP address is [REDACTED]
`

**Comment by SimplicityGuy** [comment]
The `forwarding_port` file is never written, and the API returns `0`.

**Comment by qdm12** [comment]
Fixed in ff56857fc8d777a08ebc30b1224b9c7b8ab9a9f4 in `:latest` image.

**Comment by SimplicityGuy** [comment]
Verified! Thanks for addressing this so quickly.
